### ISSUE TITLE
Z-Way  - my first "Hello Homekit" 

### ISSUE THREAD

**Issue by cflurin** [nfarina/homebridge#201]
@SphtKr: I've plug in a RazBerry into one RPi2 (BTW running node 4.1.1) and started with Z-Way.
It did the job at first go, awesome. Thanks.
My first included device is a Fibaro FGS222. It works properly but in the Z-Way Expert UI > Configuration > Association I've got an error (warning):
```
 Device changed configuration but some values are not written. Wait for device to wake up.
```
Could you please indicate a source (link) where I can investigate about this issue?

**Comment by SphtKr** [comment]
Sure, you may want to ask over at [the RaZBerry forums](http://forum.z-wave.me/viewforum.php?f=3422&sid=a92004d923fba8e9128389a65cfa8743), as I'm guessing this isn't specific to Homebridge--do you see the same kind of error when using the device via the ZWay web UI (e.g. http://your.rpi.ip:8083/ )? And/or does it work properly when you control it there?

Just off the top of my head that message might indicate a reception/range problem between the Fibaro and the RPi (which would be easy to happen with just one device), but I don't think I've ever seen that error specifically.

**Comment by cflurin** [comment]
Thanks, I know it isn't a homebridge issue, actually it works perfectly with homebridge as well as with UI port 8083 but this message irritates me. Let see what I can find on the internet.

**Comment by SphtKr** [comment]
Ah. For what it's worth, I checked one of my Fibaro dimmers and I have the same message. My guess is that this has something to do with the infamous ["Lutron Patent" workaround](https://community.smartthings.com/t/questions-about-the-lutron-patent-z-wave-instant-status-reporting/10270) used by many Z-Wave devices, which uses an association group to report status changes--perhaps Z-Way and the Fibaro dimmer have a problem in the implementation.

I imagine this will be zero-impact--I only noticed the message because of this thread and it's probably been there since I installed that device six months ago. But yes I can see how it would be annoying once you've noticed it.
